src/main/java/com/team766/controllers/CanSparkMaxSmartMotionLinearPIDController.java

```diff
@@ -0,0 +1,24 @@
+public class CanSparkMaxSmartMotionLinearPIDController extends ExtendableCanSparkMaxSmartMotionPIDController {
+	
+	private final double antigravPower;
+
+
+	public CanSparkMaxSmartMotionLinearPIDController(final String configName, final double absEncoderOffset, final double absEncoderOffsetForZeroEncoderUnits, final OffsetPoint first, final OffsetPoint second) {
```

Comment:
if the function definition line is too long, break into multiple lines, such as 1 parameter per line

Reply:
@JayAgra LINTER